patches/@types/react-native+0.52.1.patch

```diff
@@ -0,0 +1,15 @@
+patch-package
+--- a/node_modules/@types/react-native/index.d.ts
++++ b/node_modules/@types/react-native/index.d.ts
+@@ -8435,8 +8435,9 @@ export type Clipboard = ClipboardStatic;
+ export var DatePickerAndroid: DatePickerAndroidStatic;
+ export type DatePickerAndroid = DatePickerAndroidStatic;
+ 
+-export var Geolocation: GeolocationStatic;
+-export type Geolocation = GeolocationStatic;
++// https://github.com/DefinitelyTyped/DefinitelyTyped/issues/24573
```

Comment:
Gah, didn’t know we had this issue. This should really be worked around by getting rid of `@types/node`, because RN is not Node.

Looks like it’s being pulled in by enzyme/cheerio?? No clue why that would be a good idea of them.

```
=> Found "@types/node@8.5.1"
info Reasons this module exists
   - "enzyme#cheerio#parse5" depends on it
   - Hoisted from "enzyme#cheerio#parse5#@types#node"
```

Reply:
Enzyme discusses this a bit here: https://github.com/airbnb/enzyme/blob/master/docs/api/render.md#static-rendering-api

Reply:
I actually meant that no package should list `@types/*` packages as runtime deps. In this case it was `parse5` (didn’t read properly yesterday). You can see that in v5.1.0 they removed this https://github.com/inikulin/parse5/commit/4350f6a03dcd57dd9b3d8ba3c962ebf31a5f7f54#diff-b9cfc7f2cdf78a7f4b91a753d10865a2, so the fix would be for us to force resolution of that version instead of the one that cheerio depends on.

Reply:
Interesting: 

<img width="713" alt="screen shot 2018-09-12 at 12 37 59 pm" src="https://user-images.githubusercontent.com/236943/45448980-c8794b80-b688-11e8-920b-349338362b0c.png">

I removed the patch, and updated the resolution, yet the error re-appears.

Reply:
Can you do `yarn why @types/node` instead?

Reply:
<img width="702" alt="screen shot 2018-09-12 at 12 43 39 pm" src="https://user-images.githubusercontent.com/236943/45449268-87ce0200-b689-11e8-9c1e-e1eb28d398fa.png">